tests/FSharp.Test.Utilities/FSharp.Test.Utilities.fsproj

```diff
@@ -10,7 +10,8 @@
     <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
     <Optimize>false</Optimize>
     <Tailcalls>false</Tailcalls>
-    <IsTestProject>false</IsTestProject>
+    <UnitTestType>xunit</UnitTestType>
+    <IsTestProject>true</IsTestProject>
```

Comment:
Right now it would not be part of the CI run - was that intentional or not?
Is this meant more for local debugging (I could understand that perspective as well) in VS?

Reply:
Ah, yes, my oversight. It runs in CI only as part of coreClr (only because it does `dotnet test FSharp.sln`). I don't know if it's worth it to run it through the whole matrix, but maybe for completeness we should. I have no preference.

Reply:
coreClr testing is fine for me, let's keep it at that.